### PR TITLE
Fixes to debian files

### PR DESCRIPTION
**Pull request by OYTIS**
I was using `cryptodev-debian` branch to create the debian package. Now to build them in CI I'd like to merge these changes to our default branch. We will then make tags off this branch and deploy from Travis.

Note that `changelog` is static, we will either have to fix that, or update the version in it before creating tags.

### CODE REVIEW

File: build_all/packaging/linux/debian/control [hunk 1]
```diff
@@ -1,27 +1,30 @@
-Source: azure-iot-sdk-c
+Source: azure-iot-sdk-c-twilio
 Priority: optional
 Maintainer: Azure User <aziotsdklinux@outlook.com>
-Build-Depends: debhelper (>= 9), cmake (>= 2.8), libssl-dev, libcurl4-openssl-dev, curl, azure-c-shared-util-dev, azure-uamqp-c-dev, azure-umqtt-c-dev
+#Build-Depends: debhelper (>= 9), cmake (>= 2.8), libssl-dev, libcurl4-openssl-dev, curl, azure-c-shared-util-twilio-dev, azure-uamqp-c-twilio-dev, azure-umqtt-c-twilio-dev
```
Comment (rbeiter) [inline build_all/packaging/linux/debian/control:4]: Are these needed anymore?

File: build_all/packaging/linux/debian/changelog [hunk 2]
```diff
@@ -0,0 +1,5 @@
+azure-iot-sdk-c-twilio (1.0.0.1buster) buster; urgency=low
+
+  * See https://github.com/Azure/azure-iot-sdk-c/releases for details of the release
```
Comment (rbeiter) [inline build_all/packaging/linux/debian/changelog:3]: Should this point to our fork or is the overall azure--iot-sdk-c link the right spot?

**Review by rbeiter — APPROVED** [review]

File: build_all/packaging/linux/debian/changelog (hunk 2 above)
Comment (OYTIS) [reply build_all/packaging/linux/debian/changelog:3]: This is now generated automatically and looks like

```
azure-iot-sdk-c-twilio (${version}) buster; urgency=low
  * See tag ${git_branch} on https://github.com/twilio/Breakout_Trust_Onboard_SDK for source
 -- Anton Gerasimov <agerasimov@twilio.com> $(date -R)
```

File: build_all/packaging/linux/debian/control (hunk 1 above)
Comment (OYTIS) [reply build_all/packaging/linux/debian/control:4]: Probably not. As we are not doing source releases, `Build-Depends` are of little value for us, we know what should be in the build environment anyway.

**Pull request merged**